data/org.freedesktop.portal.InputCapture.xml

```diff
@@ -56,44 +56,55 @@
       This documentation describes version 1 of this interface.
    -->
   <interface name="org.freedesktop.portal.InputCapture">
-    <!--
-        CreateSession:
-        @parent_window: Identifier for the application window, see :doc:`window-identifiers`
+     <!--
+        CreateDraftSession:
         @options: Vardict with optional further information
-        @handle: Object path for the :ref:`org.freedesktop.portal.Request` object representing this call
+        @session_handle: Object path for the created :ref:`org.freedesktop.portal.Session` object
```

Comment:
This may be intentional or it may be a misunderstanding how the requests work, I'm not sure :) Are you intentionally making this *not* go through a Request here? 

Makes sense since there's no reason this can fail or be delayed so requests aren't necessary but let's make sure it's spelled out in the commit message at least.

@jadahl - do you prefer this via Request for consistency with the others?

Reply:
I literally have no idea what i am doing. I was told on the matrix that support was as easy as copying over the methods used in remote desktop. But after i was told here in review to add these new methods etc.. So im just trying todo what was asked to get this working so we can use clipboard with deskflow, input-leap etc... 

If you or anyone who knows what they are doing wants to take this over im ok with that also . Otherwise ill keep chugging along until it gets there (just maybe alot more questions along the way)

Reply:
I don't think we need to go through Request when creating a draft session.